.buildkite/pipeline.yml

```diff
@@ -26,19 +26,14 @@ steps:
   # Lint
   #################
   - label: "🧪 Lint Swift"
-    command: |
-      make lint-swift
-    plugins: *common_plugins
+    command: swiftlint
```

Comment:
Commenting on your `notify: … github_commit_status` that you removed since, to give you some more context:

 - I don't mind having clearer and more explicit GitHub Commit Status (I actually like it), but those come **in addition** to the ones auto-generated by Buildkite.
 - For example, note how we have now **both** a commit status check named `SwiftLint` (from this explicit `notify`) **and** still a commit status check named `buildkite/hostmgr/lint-swift`.

<img width="819" alt="image" src="https://github.com/Automattic/hostmgr/assets/216089/7ffb5d8e-b753-4432-9b85-85ead14d5c90">

We **_can_** disable the auto-generated commit status created by Buildkite for this pipeline (by setting `publish_commit_status = true` [here](https://github.com/Automattic/buildkite-ci/blob/trunk/src/buildkite/pipelines/hostmgr.tf#L43) if we wanted to, but then that means we'd have to ensure to add a `notify: { github_commit_status: { context: … } }` block explicitly to all the steps we have in our `pipeline.yml`, otherwise they won't be reported as commit status anymore, if not notified explicitly and we disabled implicit status as well. So it's a balance to find.

See this RFC P2 for more info: paaHJt-50A-p2

Reply:
> I don't mind having clearer and more explicit GitHub Commit Status (I actually like it), but those come in addition to the ones auto-generated by Buildkite.

Exactly 🎯 that's why I removed them (they're quite "in your face" in this repo as there aren't many jobs); also, the repo "required check" setting is already using the auto-generated ones, and they're kinda clear enough?

> We can disable the auto-generated commit status created by Buildkite for this pipeline

Too bad this is setting is on the pipeline and not at the job level: it would be handy to just prevent the auto-generated one in these cases.

Reply:
> Too bad this is setting is on the pipeline and not at the job level: it would be handy to just prevent the auto-generated one in these cases.

💯 agree.
I think this could be a good think to post about [in the "Feature Requests" section of Buildkite's forums](https://forum.buildkite.community/c/feature-requests/11) 😉 
I [already see a couple of topics about GitHub commit status generated names](https://forum.buildkite.community/search?q=github%20status), but didn't see one about disabling the generated one per-step if a custom one was set on said step…

Reply:
Feature Request created: https://forum.buildkite.community/t/allow-disabling-generated-github-commit-statues-per-step/3766